### DISCUSSION TITLE
how consume for backlog message

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#19723] [Q&A] [answered]
May I ask if "Producer is blocked on creation because backlog exceeded on topic" is triggered because of limitSize, and the consumer does not ACK for other reasons? What about the backlog of messages? In addition, no new messages can be produced in this case

**Comment by pzs-pzs** [comment, marked as answer]
Maybe you can skip some backlog msg, just read the doc https://pulsar.staged.apache.org/docs/en/admin-api-topics/#skip-messages
